README.md

```diff
@@ -5,19 +5,17 @@ Comments regarding this document are welcome. Please file issues and PRs directl
 
 Editors:
 - Caspar Roelofs, Founder/Director | Gimly caspar@gimly.io
-- Jack Tanner, Blockchain and SSI Developer | Gimly jack@gimly.io
+- Jack Tanner, Blockchain and SSI Developer | Tonomy jack@tonomy.foundation
 - Markus Sabadello | Danube Tech markus@danubetech.com
 
-TODO: turn into a Re-spec page and host https://respec.org/docs
-
 # Introduction
 
-VerifiableCondition is a new type of verification method for DID Documents. It can be used to express complex conditions and additional meta data about verification methods. It can be used to combine verification methods together to form conjugated conditions such as logical operations &&, thresholds, weighted thresholds, relationships and a delegation to external verification methods.
+ConditionalProof is a new type of verification method for DID Documents. It can be used to express complex conditions and additional meta data about verification methods. It can be used to combine verification methods together to form conjugated conditions such as logical operations &&, thresholds, weighted thresholds, relationships and a delegation to external verification methods.
```

Comment:
```suggestion
`ConditionalProof` is a new type of verification method for DID Documents. It can be used to express complex conditions and additional metadata about verification methods. It can be used to combine verification methods to form conjugated conditions such as logical operations (like `&&`), thresholds, weighted thresholds, relationships, or delegation to external verification methods.
```